docs/spikes/azure-sphere.md

```diff
@@ -0,0 +1,90 @@
+# What is Azure Sphere
+
+[Azure Sphere](https://azure.microsoft.com/en-us/services/azure-sphere/) is a security service
+combining hardware, OS-level software, and cloud services to secure IoT applications. Together these
+three components ensure [The Seven Properties of Highly Secure
+Devices](https://www.microsoft.com/en-us/research/uploads/prod/2020/11/Seven-Properties-of-Highly-Secured-Devices-2nd-Edition-R1.pdf).
+
+[Pricing](https://azure.microsoft.com/en-us/pricing/details/azure-sphere/): one time purchase of
+hardware only (no recurring fees)
+
+## Advantages for the starter kit
+
+- Automated management of the underlying OS and the concentrator logic.
+- Although some features are already supported e.g. password-less authentication, we could benefit
+  from built-in, e.g:
+  - hardware security
+  - error reporting (i.e. A software error, such as a buffer overrun induced by an attacker probing
+security, is reported to cloud-based failure analysis system)
+  - renewable security (revoking compromised assets for known vulnerabilities)
```

Comment:
I don't understand this point, I think it would be worth expanding a bit on what "renewable security" is and what "revoking compromised assets" means.

Reply:
Good point, simplified it to "automated security updates" instead